framework/include/transfers/MultiAppConservativeTransfer.h

```diff
@@ -0,0 +1,69 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "MultiAppFieldTransfer.h"
+
+// Forward declarations
+class MultiAppConservativeTransfer;
+
+template <>
+InputParameters validParams<MultiAppConservativeTransfer>();
+
+/**
+ *  This serves an interface for MultiAppInterpolationTransfer, MultiAppNearestNodeTransfer and so
+ * on.
```

Comment:
Well that's not really what this class does. It conserves the "total" value of a quantity being transferred.